### ISSUE TITLE
Error syncing BazQux

### ISSUE THREAD

**Issue by aredridel** [Ranchero-Software/NetNewsWire#3605]
In Console, I see `Refresh missing articles failed: <private>.` frequently. The `<private>` seems to be where a description of the error goes if I am reading the code right.

The net effect is that nothing syncs from BazQux at all — my feeds show as read, even though most of them have articles.

The complete log in console for a given task doesn't seem to show any error, but the above error usually appears after the "sent request, body" line, but does not mention the task UUID:

```
default	15:51:11.365446-0400	NetNewsWire	Task <C2A224FE-BE26-45FD-8EF1-6108C21EB0AF>.<69> resuming, timeouts(60.0, 604800.0) QOS(0x21) Voucher <private>
default	15:51:11.366270-0400	NetNewsWire	[Telemetry]: Activity <nw_activity 12:2 [45606129-EFE7-4CE9-9E20-8B4FCC59B107] (reporting strategy default)> on Task <C2A224FE-BE26-45FD-8EF1-6108C21EB0AF>.<69> was not selected for reporting
default	15:51:11.366854-0400	NetNewsWire	Task <C2A224FE-BE26-45FD-8EF1-6108C21EB0AF>.<69> {strength 0, tls 4, ct 0, sub 0, sig 1, ciphers 0, bundle 0, builtin 0}
default	15:51:12.789172-0400	NetNewsWire	Task <C2A224FE-BE26-45FD-8EF1-6108C21EB0AF>.<69> now using Connection 32
default	15:51:12.793119-0400	NetNewsWire	Task <C2A224FE-BE26-45FD-8EF1-6108C21EB0AF>.<69> sent request, body S 7168
default	15:51:13.113922-0400	NetNewsWire	Task <C2A224FE-BE26-45FD-8EF1-6108C21EB0AF>.<69> received response, status 200 content C
default	15:51:13.466406-0400	NetNewsWire	Task <C2A224FE-BE26-45FD-8EF1-6108C21EB0AF>.<69> response ended
default	15:51:13.466511-0400	NetNewsWire	Task <C2A224FE-BE26-45FD-8EF1-6108C21EB0AF>.<69> done using Connection 32
default	15:51:13.466685-0400	NetNewsWire	Task <C2A224FE-BE26-45FD-8EF1-6108C21EB0AF>.<69> summary for task success {transaction_duration_ms=2099, response_status=200, connection=32, reused=1, request_start_ms=1423, request_duration_ms=2, response_start_ms=1746, response_duration_ms=353, request_bytes=7600, response_bytes=117084, cache_hit=false}
default	15:51:13.467185-0400	NetNewsWire	Task <C2A224FE-BE26-45FD-8EF1-6108C21EB0AF>.<69> finished successfully
```

**Comment by vincode-io** [comment]
@aredridel have you tried removing and adding your BazQux account?

I'm not able to recreate the problem. If adding and removing the account doesn't work, could you supply an OPML file from your account? If you don't feel comfortable sharing it on GitHub, you can email it to me directly. My email is mo@vincode.io.

**Comment by aredridel** [comment]
I did indeed, and it was fine for a few days — I'd figured it was the intermittent bazqux credential problem fixed earlier until I dug into it.

OPML coming at you by email! Feel free to put anything that turns out to be relevant on the ticket, but I'd love to avoid posting the whole thing.

**Comment by vincode-io** [comment]
I was able to import the OPML file you sent and don't have any problems syncing with it. The problem might be with an article that I don't have. Do you have a huge number of unread articles? We maybe hitting some limitation there.

**Comment by aredridel** [comment]
So it worked for a few minutes, then stopped again later in the day. Removing one 500+ article/day feed, however, and everything works great.

**Comment by vincode-io** [comment]
@aredridel Thanks for following up. If you feel comfortable doing so, could you please share the article of the feed that was causing the problem?

**Comment by aredridel** [comment]
https://o-blivia.tumblr.com/rss was the culprit

**Comment by vincode-io** [comment]
@aredridel I do have that feed in my BazQux account and it isn't causing any problems for me (so far).

What happens if you add it back in again?

**Comment by aredridel** [comment]
@vincode-io it might have to do with my habits — I routinely close NNW for long enough that more than 500 posts accumulate in the feed.

I'll add it back and see!

**Comment by vincode-io** [comment]
BazQux has a 500 article limit. It might be that they had a bug and that this feed ended up with more in it than was it was supposed to and that caused the problem. I'm just wildly speculating tho...

**Comment by aredridel** [comment]
Nope! NNW won't update. here's the state in BQ:

<img width="457" alt="Screen Shot 2022-07-30 at 4 15 19 PM" src="https://user-images.githubusercontent.com/2876/181994871-6b40b949-b789-423e-850c-33d4d3d4dc88.png">

And NNW after a refresh says there are no unread articles.

**Comment by aredridel** [comment]
But after deleting it again things aren't refreshing. Maybe this is a red herring? Something's funky though.

**Comment by vincode-io** [comment]
@aredridel Do you know how to use ProxyMan to snoop network traffic? We might be getting a server return code that will tell us something.

**Comment by aredridel** [comment]
Yeah, I can do that!

Also, adding that feed back killed it; removing the bazqux account and re-adding it got it to sync again.

**Comment by vincode-io** [comment]
I'm pretty sure that this was resolved with the fix in #3708. If we still encounter it after the next release, we can reopen this ticket to investigate more.
